keps/sig-storage/1698-generic-ephemeral-volumes/kep.yaml

```diff
@@ -0,0 +1,30 @@
+title: generic ephemeral inline volumes
+kep-number: 1698
+authors:
+  - "@pohly"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+status: provisional
```

Comment:
Is this ready to be updated to implementable?

Reply:
This to go in as an alpha next or beta?

Reply:
I hope so. Let me make the change, otherwise it will miss 1.19.